packages/utils/src/add-static-fields.ts

```diff
@@ -0,0 +1,15 @@
+import type { FC } from 'react';
+
+type StaticFields = Record<string, unknown>;
+
+export default function addStaticFields<
+  Props extends {},
```

Comment:
Should we prefix types with T or not. I am open to either way. It helped me to have them prefixed but that is subjective.

Reply:
Yeah I also don't have strong opinions. I think I tend to prefix them if the types are exported.

Reply:
I used to have the opinion of prefixing them, but their role/purpose is becoming less meaningful since we have intellisense.

Since this is open source, I'd add the prefix to increase readability, especially for those who don't have intellisense (for whatever reason). The same principle applies to why we suffix function types with `*Fn` or lists with `*List` (list is not a type, but the suffix indicates a set or array)

what do you think? 

---

note: I think we should drop this component eventually
it is used in a few instance and none on MC if I recall correctly. best to double check

Reply:
Yeah. It's not like I made up that T-prefix. I saw it in Apollo and react-ui where it helped me reading the code knowing later what a type is and what a variable.

Reply:
now that you mentioned it; another "beauty" with adding the prefix is that you can identify a type/enum/whateverz when browsing in github. that's a bonus.

Reply:
pushing a commit to fix this so we can merge sooner 😝